tests/data/ap-waf/syslog.yaml

```diff
@@ -29,4 +37,34 @@ spec:
       targetPort: 514
       protocol: TCP
   selector:
-    app: syslog
\ No newline at end of file
+    app: syslog
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: syslog-config
+data:
+  syslog-ng.conf: |-
+    @version: 3.29
```

Comment:
just curious, what are the reasons for adding the config?

Reply:
the logs were saying that the connection was refused because the max number of connections was reached. So that is basically the default config from the container and I added max_connections